gatsby_fp/.nvmrc

```diff
@@ -0,0 +1 @@
+14.17.0
```

Comment:
Why would we use node 14.17.0 and not the stable version 14.1.0?
https://www.gatsbyjs.com/docs/reference/release-notes/migrating-from-v2-to-v3/#minimal-nodejs-version-12130

Reply:
Please provide a link to something that says 14.1.0 is more stable than 14.17.0.

Based on https://github.com/nodejs/Release#nodejs-release-working-group it seems like "14.x" is active LTS and typically "x" means the highest number available.

Reply:
Good catch. For some reason, when I ran `nvm list` on my local machine, I got `stable -> 14.1 (-> v14.1.0) (default)`.
But https://nodejs.org/en/ is where I should have looked first, which shows 14.7.4.

Reply:
Or I should've run `nvm ls-remote` which shows the list of available node versions on remote.